src/app/item-card/item-card.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="item">
+  <div class="__container">
+    <img class="__item-image" [src]="itemData.image" />
+    <div class="__item-text">
+      <h4 class="__item-title">{{ title }}</h4>
+      <p>{{ description }}</p>
+      <div class="__price-fav">
+        <h5>
+          {{ itemData.price.amount | currency: itemData.price.currency }}
+        </h5>
+        <div [ngSwitch]="itemData.favorited">
```

Comment:
Do you think is worth using a switch when we only have 2 conditions?

Reply:
No, but I thought it could be a nice moment to apply in the project to test this angular's element. Do you think I should keep it just to use this feature or better to change it?